docs/src/pages/components/cdk-components/switch/api.jsx

```diff
@@ -59,13 +59,14 @@ const switchPropsTable = () => {
         <td>If true, the component will be disabled.</td>
       </tr>
       <tr>
-        <td>required: boolean</td>
+        <td>optional: boolean</td>
         <td>
           <code>false</code>
         </td>
         <td>
-          If true, the switch will change its appearence, showing that the value
-          is required.
+          If true, the component will be optional, showing{" "}
+          <code>(Optional)</code> next to the label. Otherwise, it should be
+          considered required and display an error if it is convenient.
```

Comment:
Maybe this is not correct since the component has no way of showing errors

Reply:
That's why I say "...if it is convenient." and "...it should be considered..." in the prop description. Maybe I need to rephrase it like this:

- If true, the component will be optional, showing `(Optional)` next to the label. Otherwise, it should be considered required and, if it is convenient, display an error message.

What do you think?